src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/lib/embeddable/dashboard_container_factory.ts

```diff
@@ -18,19 +18,29 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { SavedObjectMetaData } from 'ui/saved_objects/components/saved_object_finder';
-import { SavedObjectAttributes } from 'src/core/server';
+import { SavedObjectAttributes } from '../../../../../../../../core/server';
```

Comment:
Why the change away from `src/core/server`?  If memory serves, that is the latest recommended way to import (switched from relative to that, was my understanding)

Reply:
I had CI checks failing when importing `src/...` from some locations. I can't say if this one was one of them. Also, I commented out saved object logic for quite some time, when I put it back, maybe this is how VSCode auto-imported it.